src/components/page-container.tsx

```diff
@@ -92,7 +92,7 @@ function PageContainer(props: PageContainerProps) {
                       </Badge>
                     )}
                     {children}
-                    <Box mt='40px'>
+                    <Box mt='40vh'>
```

Comment:
This change causes every page to have a huge whitespace at the bottom. This should not happen.